### DISCUSSION TITLE
q-stepper not going back to top when doing a next()

### DISCUSSION THREAD

**Discussion by amasson84** [quasarframework/quasar#9551] [General - Components / Directives / etc]
I have a weird bug with the project I'm working one
I'm using q-stepper and when I do next() with chrome the new page is not on the top but where the next button was on the previous page
This bug doesn't exist with Firefox, apparently.
Someone has a clue how to fix this?

**Comment by metalsadman** [comment]
Make a codepen.

**Reply by amasson84** [reply]
I tried : https://jsfiddle.net/fx7b465o/
But the project is too complex to export.
Using fiddle both firefox and chrome don't go to top when clicking on next...

**Reply by ontwikkelfabriek** [reply]
@amasson84 
> This bug doesn't exist with Firefox, apparently.

Why would this be bug?  Not everyone wants to scroll to the top of their stepper. 


I changed this code in your jsfiddle and it scrolls to the top when you press next.

```javascript
<q-btn class="col-auto" @click="onNext" color="primary" label="NEXT" />
```

```javascript
onNext() {
   this.$refs.stepper_obj.next()
   Vue.nextTick(function () {
      this.window.scrollTo(0,0);
   })
      
}
```

**Comment by kensoz** [comment]
I meet a same bug in chrome with vue3
I try to use `window.scrollTo(0,0)` after `stepper.next() ` and `stepper.previous()`
it was successed

but after `stepper.goTo()` 
it need to add ` window.addEventListener('scroll',  xxx)` to watch the scroll change
bacause this api will change scroll twice!!

**Reply by pdanpdan** [reply]
Can you please make a codepen?

**Reply by kensoz** [reply]
[codepen](https://codepen.io/kensoz/pen/zYPGRqV)

I am sorry I can not make a `stepper.goTo()` reproduction
in my project code is complex (typescript, async ...etc)
but this codepen show not going back to top when `stepper.next()` and `stepper.previous()`
I will check my project and I like quasar
